robotpy_installer/installer.py

```diff
@@ -1084,6 +1091,21 @@ def install_opkg_opts(self, parser):
             "--ignore-image-version", action="store_true", default=False
         )
 
+    def _load_opkg_from_req(self, *files):
+        """
+            Pull the list of opkgs from the files
+        """
+        opkgs = []
+        # Loop through the passed in files to support multiple requirements files
+        for file in files:
+            with open(file, "r") as f:
+                for row in f.readlines():
+                    # Ignore commented lines and empty lines
+                    if row.strip() and not row.startswith("#"):
```

Comment:
Why not strip once before the if statement instead of stripping twice?

Reply:
That would probably make more sense. Original thought was to preserve file formatting but also could be slightly bugged if there's a space before the #. And also the formatting of the file doesn't really matter when it's being parsed.